aip/0135.md

```diff
@@ -179,6 +179,30 @@ message DeleteBookRequest {
 If the etag is provided and does not match the server-computed etag, the
 request **must** fail with a `FAILED_PRECONDITION` error code.
 
+### Deletion prevention
+
+Sometimes, it may be desirable for users to protect a resource against accidental
+deletion. This is often especially useful in "infrastructure as code" deployments
+where resources are kept in sync with some declarative configuration.
+
+To achieve this, the resource **should** include a `bool prevent_delete` field to
```

Comment:
`deletion_protection` is a phrasing that's already been used in [at least one](https://cloud.google.com/compute/docs/reference/rest/v1/instances#Instance.FIELDS.deletion_protection) GCP API. I know AIPs aren't just for GCP, but unless we're seeing `prevent_delete` in other places already, would it make sense to call it `deletion_protection` for some backwards consistency?

Reply:
The same reason we use `create_time` and not `creation_time` is the justification AFAICT.